backend/api/src/reaction.ts

```diff
@@ -110,8 +110,8 @@ export const addOrRemoveReaction: APIHandler<'react'> = async (props, auth) => {
           .update({ likes: count ?? 0 })
           .eq('comment_id', contentId)
         
-        // Broadcast the updated comment
-        broadcastCommentUpdate(contentId, { comment_id: contentId, likes: count ?? 0 })
+        // Broadcast the new like
+        broadcastNewLike(contentId, { comment_id: contentId, likes: count ?? 0 })
```

Comment:
Instead of sending likes, you should broadcast a single reaction object, which is one instance of a like, and includes the user id and content id. The client tallies them to get the total.

You should modify the client to receive this data correctly.